third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -268,6 +268,10 @@ region are guaranteed to support the same version.
 * `private_cluster_config` - (Optional) Configuration for [private clusters](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters),
 clusters with private nodes. Structure is documented below.
 
+* `cluster_telemetry` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)) Configuration for
+   [ClusterTelemetry](https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#Cluster.ClusterTelemetry) feature,
```

Comment:
```suggestion
   [ClusterTelemetry](https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#Cluster.ClusterTelemetry) feature.
```

Is there a GKE docs page that describes the feature in a more user-friendly way than the API page, or is that it for docs?

Reply:
I pointed to this page. This is the closest i see, https://cloud.google.com/monitoring/kubernetes-engine/installing#controlling_the_collection_of_application_logs